plugins/extractors/elastic/elastic.go

```diff
@@ -101,22 +102,32 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) (err error)
 		return
 	}
 	x := reflect.ValueOf(r["indices"]).MapRange()
+	var indexes []string
 	for x.Next() {
 		indexName := x.Key().String()
-		docProperties, err1 := e.listIndexInfo(x.Key().String())
+		indexes = append(indexes, indexName)
+	}
+	sort.Strings(indexes)
```

Comment:
does the indices order also cause the flaky test, or is it just columns order?

Reply:
yes there were some instance where it was out of order